robot_skills/src/robot_skills/arms.py

```diff
@@ -243,38 +264,52 @@ def send_joint_goal(self, configuration, timeout=5.0):
             rospy.logwarn('Default configuration {0} does not exist'.format(configuration))
             return False
 
-    def send_joint_trajectory(self, configuration, timeout=5.0):
-        '''
+    def send_joint_trajectory(self, configuration, timeout=5):
+        """
         Send a named joint trajectory (sequence of poses) defined in the default_trajectories to
         the arm
-        '''
+        :param configuration: name of configuration, configuration should be loaded as parameter
+        :param timeout: timeout in seconds
+        :return: True or False, False in case of nonexistent configuration or failed execution
+        """
         if configuration in self.default_trajectories:
             return self._send_joint_trajectory(self.default_trajectories[configuration], timeout=rospy.Duration(timeout))
         else:
             rospy.logwarn('Default trajectories {0} does not exist'.format(configuration))
             return False
 
     def reset(self, timeout=0.0):
-        '''
+        """
         Put the arm into the 'reset' pose
-        '''
+        :param timeout: timeout in seconds
+        :return: True or False
+        """
         return self.send_joint_goal('reset', timeout=timeout)
 
     @property
     def occupied_by(self):
-        '''
+        """
         The 'occupied_by' property will return the current entity that is in the gripper of this arm.
-        '''
+        :return: robot_skills.util.entity, ED entity
+        """
         return self._occupied_by
 
     @occupied_by.setter
     def occupied_by(self, value):
+        """
+        Set the entity which occupies the arm.
+        :param value: robot_skills.util.entity, ED entity
+        :return: no return
+        """
         self._occupied_by = value
 
     def send_gripper_goal(self, state, timeout=5.0):
-        '''
+        """
         Send a GripperCommand to the gripper of this arm and wait for finishing
-        '''
+        :param state: open or close
+        :param timeout: timeout in seconds; timeout of 0.0 is not allowed
+        :return: True of False
+        """
         goal = GripperCommandGoal()
 
         if state == 'open':
```

Comment:
Comparing to a member of GripperState (used to be ArmState) for consistency is nice